code/modules/scrap/scrap.dm

```diff
@@ -429,7 +429,7 @@ GLOBAL_LIST_EMPTY(scrap_base_cache)
 	loot_max = 10
 	rarity_value = 90
 	loot_tags = list(
-		SPAWN_GUN = 0.3,
+		SPAWN_GUN_HANDMADE = 0.3,
 		SPAWN_PART_GUN = 3,
```

Comment:
Consider decreasing SPAWN_GUN_HANDMADE to ~0.1-0.2, considering the lack of variety that spawner provides.

Reply:
I think it's fine as is, the 0.3 is really rare already.